src/javascript/Create/CreateNewContentAction/createNewContent.action.jsx

```diff
@@ -47,7 +47,9 @@ const CreateNewContent = ({context, render: Render, loading: Loading}) => {
     const {uilang, language} = useSelector(state => ({language: state.language, uilang: state.uilang}));
     const res = useNodeChecks(
         {path: context.path, language: language},
-        {...context}
+        {
```

Comment:
removing ...context might break things as it's used to pass params to useNodeChecks

Reply:
Create.actions.js actually pass the following : 
        showOnNodeTypes: ['jnt:contentFolder', 'jnt:content'],
        requiredPermission: ['jcr:addChildNodes']
So I don't think hideOnNodeTypes for jnt:navMenuText is needed .. ?